src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,23 @@
+package step2
+
+enum class Operator(val symbol: String) {
+    ADD("+") {
+        override fun execute(operand1: Int, operand2: Int): Int = operand1 + operand2
+    },
+    SUBTRACT("-") {
+        override fun execute(operand1: Int, operand2: Int): Int = operand1 - operand2
+    },
+    MULTIPLY("*") {
+        override fun execute(operand1: Int, operand2: Int): Int = operand1 * operand2
+    },
+    DIVIDE("/") {
+        override fun execute(operand1: Int, operand2: Int): Int = operand1 / operand2
+    };
+
+    companion object {
+        fun bySymbol(symbol: String): Operator? {
+            return Operator.values().find { it.symbol == symbol }
+        }
```

Comment:
코틀린의 함수는 단일 표현식일 경우 다음 코드처럼 표현할 수 있습니다!
```suggestion
        fun bySymbol(symbol: String): Operator? = Operator.values().find { it.symbol == symbol }
```